internal/model/db.go

```diff
@@ -5,13 +5,14 @@ import (
 	"fmt"
 	"log"
 	"os"
+
+	// pg sets up postgres sql
+	_ "github.com/lib/pq"
```

Comment:
Moved this to `db.go` since it was DB related. Just had to add a comment as to why to make `govet`/the-linter happy.